src/patternfly/layouts/Flex/examples/Flex.md

```diff
@@ -769,6 +769,98 @@ Ordering - Ordering can be applied to nested <code>.pf-l-flex</code> and <code>.
 {{/l-flex}}
 ```
 
+### Row gap
+```hbs
+{{#> l-flex l-flex--modifier="pf-m-row-gap" l-flex--attribute='style="--pf-l-flex--RowGap: var(--pf-global--spacer--lg);"'}}
+  {{#> l-flex l-flex--modifier="pf-m-row-gap" l-flex--attribute='style="--pf-l-flex--RowGap: var(--pf-global--spacer--sm);"'}}
+    {{#> l-flex-item}}
+      Set 1, Item A
+    {{/l-flex-item}}
+    {{#> l-flex-item}}
+      Set 1, Item B
+    {{/l-flex-item}}
+    {{#> l-flex-item}}
+      Set 1, Item C
+    {{/l-flex-item}}
+    {{#> l-flex-item}}
+      Set 1, Item D
+    {{/l-flex-item}}
+    {{#> l-flex-item}}
+      Set 1, Item E
+    {{/l-flex-item}}
+    {{#> l-flex-item}}
+      Set 1, Item F
+    {{/l-flex-item}}
+  {{/l-flex}}
+  {{#> l-flex l-flex--modifier="pf-m-row-gap" l-flex--attribute='style="--pf-l-flex--RowGap: var(--pf-global--spacer--lg); --pf-l-flex--RowGap-on-xl: var(--pf-global--spacer--2xl);"'}}
```

Comment:
this is responsive - it reads like the example under it has the responsive examples.